README.md

```diff
@@ -20,29 +20,31 @@ Contents are added frequently! ⚡
 ---
 ## Table of Contents
 
-- [Blockchain](#blockchain)
-- [Cyber Security](#cyber-security)
-- [Data Science](#data-science)
-- [Design](#design)
-- [Developer Tools](#developer-tools)
-- [DevOps](#devops)
-- [Economics](#economics)
-- [Education](#education)
-- [Finance](#finance)
-- [Gaming](#gaming)
-- [Health](#health)
-- [Internet of Things](#internet-of-things)
-- [Machine Learning](#machine-learning)
-- [Marketing](#marketing)
-- [Mobile](#mobile)
-- [Productivity](#productivity)
-- [Programming](#programming)
-- [Project Management](#project-management)
-- [Science](#science)
-- [Software Engineering](#software-engineering)
-- [Startups](#startups)
-- [Thanks to all Contributors 💪](#thanks-to-all-contributors-)
-- [Thanks to all Stargazers ⭐️](#thanks-to-all-stargazers-️)
+- [Awesome Article Collection](#awesome-article-collection)
+  - [Table of Contents](#table-of-contents)
```

Comment:
I think you have some MD extensions that auto-added additional bullet points here; kindly review them.